sdk/keyvault/Azure.Security.KeyVault.Secrets/src/Azure.Security.KeyVault.Secrets.csproj

```diff
@@ -26,5 +26,6 @@
   <ItemGroup>
     <Compile Include="$(AzureCoreSharedSources)PageResponseEnumerator.cs" />
     <Compile Include="$(AzureCoreSharedSources)ArrayBufferWriter.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\Shared\ChallengeBasedAuthenticationPolicy.cs" />
```

Comment:
I don't think you need `$(MSBuildThisFileDirectory)` Includes are relative to project directory by default